README.rst

```diff
@@ -208,45 +235,43 @@ releases. You need to register a username and password with both these sites.
 The username and passwords for both these sites need not be the same. To upload
 your package through the command line, you need to store your username and
 password in a file called `.pypirc` in your `$HOME` directory with the
-following format.
-
-```shell
-[distutils]
-index-servers =
-  pypi
-  pypitest
-
-[pypi]
-repository=https://pypi.python.org/pypi
-username=<your-pypi-username>
-password=<your-pypi-passowrd>
-
-[pypitest]
-repository=https://testpypi.python.org/pypi
-username=<your-pypitest-username>
-password=<your-pypitest-passowrd>
-```
+following format::
+
+  [distutils]
+  index-servers =
+    pypi
+    pypitest
+
+  [pypi]
+  repository=https://pypi.python.org/pypi
+  username=<your-pypi-username>
+  password=<your-pypi-passowrd>
+
+  [pypitest]
+  repository=https://testpypi.python.org/pypi
+  username=<your-pypitest-username>
+  password=<your-pypitest-passowrd>
+
 Make sure that all details in `setup.py` are up to date. To upload your package
-to the Test server, execute:
-```
-python setup.py register -r pypitest
-python setup.py sdist upload -r pypitest
-```
+to the Test server, execute::
+
+  $ python setup.py register -r pypitest
+  $ python setup.py sdist upload -r pypitest
+
 Your package should now be visible on: https://testpypi.python.org/pypi
 
 To install a package from the test server, execute:
-```
-pip install -i https://testpypi.python.org/pypi <package-name>
-```
+
+  $ pip install -i https://testpypi.python.org/pypi <package-name>
+
 
 Similary, to upload your package to the PyPI server execute
-```
-python setup.py register -r pypi
-python setup.py sdist upload -r pypi
-```
+
+  $ python setup.py register -r pypi
+  $ python setup.py sdist upload -r pypi
```

Comment:
This is an outdated way of uploading to PyPi. Now it is recommended to use twine.

I could remove the contents of this section altogether and link to https://packaging.python.org/tutorials/packaging-projects/ (by PyPA) instead -- keeping these best practices up to date is what is it for.